src/components/problemDetails/ProblemDetailView.tsx

```
@@ -0,0 +1,318 @@
+                        <EyeOff className="h-4 w-4 mr-1" />
+                      ) : (
+                        <Eye className="h-4 w-4 mr-1" />
+                      )}
+                      {expandedDescription ? 'Collapse' : 'Expand'}
+                    </Button>
+                  </div>
+                  <div
+                    className={`text-sm text-muted-foreground prose prose-sm max-w-none ${
+                      !expandedDescription && 'line-clamp-3'
+                    }`}
+                    dangerouslySetInnerHTML={{ __html: problem.problem.description }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)